TowerForge/TowerForge/GameModule/GameModes/DeathMatchMode.swift

```diff
@@ -45,14 +45,18 @@ class DeathMatchMode: GameMode {
     }
     func updateGame(deltaTime: TimeInterval) {
         self.remainingTime = timer.time
-        if remainingTime <= 0 {
-            if currentOwnKillCounter > currentOpponentKillCounter {
-                gameState = .WIN
-            } else if currentOwnKillCounter < currentOpponentKillCounter {
-                gameState = .LOSE
-            } else {
-                gameState = .DRAW
-            }
+        guard gameState != .LOSE else {
+            return
+        }
+        guard remainingTime < 0 else {
```

Comment:
hmm why is this a `return`

Reply:
So that we don't override the concede when updateGame is called.